.buildkite/pipelines/es_serverless/verify_es_serverless_image.yml

```diff
@@ -144,11 +144,113 @@ steps:
         parallelism: 1
         retry:
           automatic:
-            - exit_status: "-1"
+            - exit_status: '-1'
+              limit: 1
+
+      - command: .buildkite/scripts/steps/functional/security_serverless_rule_management_prebuilt_rules_installation.sh
+        label: 'Serverless Rule Management - Prebuilt Rules Installation - Security Solution Cypress Tests'
+        if: "build.env('SKIP_CYPRESS') != '1' && build.env('SKIP_CYPRESS') != 'true'"
+        agents:
+          image: family/kibana-ubuntu-2004
+          imageProject: elastic-images-prod
+          provider: gcp
+          machineType: n2-standard-4
+          preemptible: true
+        depends_on: build
+        timeout_in_minutes: 60
+        parallelism: 1
+        retry:
+          automatic:
+            - exit_status: '-1'
+              limit: 1
+
+      - command: .buildkite/scripts/steps/functional/security_serverless_rule_management_prebuilt_rules_management.sh
+        label: 'Serverless Rule Management - Prebuilt Rules Management - Security Solution Cypress Tests'
+        if: "build.env('SKIP_CYPRESS') != '1' && build.env('SKIP_CYPRESS') != 'true'"
+        agents:
+          image: family/kibana-ubuntu-2004
+          imageProject: elastic-images-prod
+          provider: gcp
+          machineType: n2-standard-4
+          preemptible: true
+        depends_on: build
+        timeout_in_minutes: 60
+        parallelism: 1
+        retry:
+          automatic:
+            - exit_status: '-1'
+              limit: 1
+
+      - command: .buildkite/scripts/steps/functional/security_serverless_rule_management_prebuilt_rules_upgrade.sh
+        label: 'Serverless Rule Management - Prebuilt Rules Upgrade - Security Solution Cypress Tests'
+        if: "build.env('SKIP_CYPRESS') != '1' && build.env('SKIP_CYPRESS') != 'true'"
+        agents:
+          image: family/kibana-ubuntu-2004
+          imageProject: elastic-images-prod
+          provider: gcp
+          machineType: n2-standard-4
+          preemptible: true
+        depends_on: build
+        timeout_in_minutes: 60
+        parallelism: 1
+        retry:
+          automatic:
+            - exit_status: '-1'
+              limit: 1
+
+      - command: .buildkite/scripts/steps/functional/security_serverless_detection_engine.sh
+        label: 'Serverless Detection Engine - Security Solution Cypress Tests'
+        if: "build.env('SKIP_CYPRESS') != '1' && build.env('SKIP_CYPRESS') != 'true'"
+        agents:
+          image: family/kibana-ubuntu-2004
+          imageProject: elastic-images-prod
+          provider: gcp
+          machineType: n2-standard-4
+          preemptible: true
+        depends_on: build
+        timeout_in_minutes: 60
+        parallelism: 5
+        retry:
+          automatic:
+            - exit_status: '-1'
+              limit: 1
```

Comment:
Is this an unrelated addition? We should backport only our changes from the base [PR](https://github.com/elastic/kibana/pull/231150).